src/Microsoft.Azure.WebJobs.Host/Extensions/Tooling.cs

```diff
@@ -0,0 +1,397 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.Azure.WebJobs.Host.Config;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    // Provides additional bookkeeping on extensions 
+    internal class Tooling : ITooling
```

Comment:
ok. It's private, so we can always change later. 

---
In reply to: [108524012](https://github.com/Azure/azure-webjobs-sdk/pull/1093#discussion_r108524012) [](ancestors = 108524012)